### PR TITLE
1.6 release announcement

### PR DESCRIPTION
**Pull request by steveklabnik**
r? @aturon 

(shouldn't be merged until thursday)

### CODE REVIEW

File: _posts/2016-01-21-Rust-1.6.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+is what it does _not_ contain: significant dependencies. `libcore` is completely
+platform agnostic, and requires only a handful of external symbols to be defined.
+Rust’s `libstd` builds on top of `libcore`, adding support for memory allocation,
+I/O, and concurrency. Applications using Rust in the embedded space, as well as
+those writing operating systems, often eschew `libstd`, using only `libcore`.
+
+[`libcore`]: http://doc.rust-lang.org/nightly/core/
+
+`libcore` being stabilized is a major step towards being able to write the lowest
+levels of software using stable Rust. There’s still future work to be done, however,
```
Comment (aturon) [inline _posts/2016-01-21-Rust-1.6.md]: Let's be a bit more specific about this: we can say that this step is enough to build a stable library ecosystem around `libcore`, but a bit more work is needed before stable Rust supports *applications* at the `libcore` level.

File: _posts/2016-01-21-Rust-1.6.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+### What's in 1.6 stable
+
+This release contains a number of small refinements, one major feature, and
+a change to [Crates.io](https://crates.io).
+
+#### libcore stabilization
+
+The largest new feature in 1.6 is that [`libcore`] is now stable! Rust’s standard
+library is two-tiered: `libcore` contains a lot of functionality, but more important
```
Comment (aturon) [inline _posts/2016-01-21-Rust-1.6.md]: I think you want to say: "... is two-tiered: there's a small core library, `libcore`, and the full standard library, `libstd`, that builds on top of it`."

File: _posts/2016-01-21-Rust-1.6.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+memory, reducing allocation in certain situations.
+
+A number of implementations of `From` for converting between standard library
+types, mainly between various integral and floating-point types.
+
+Finally, `Vec::extend_from_slice()`, which was previously known as
+`push_all()`. This method has a significantly faster implementation than the
+more general `extend()`.
+
+See the detailed release notes linked above for more.
```
Comment (aturon) [inline _posts/2016-01-21-Rust-1.6.md]: Just make this a link :-)

File: _posts/2016-01-21-Rust-1.6.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+If you maintain a crate on [Crates.io](https://crates.io), you might have seen
+a warning: newly uploaded crates are no longer allowed to use a wildcard when
+describing their dependencies. In other words, this is not allowed:
+
+```toml
+[dependencies]
+regex = "*"
+```
+
+Instead, you must actually specify a specific version or range of versions,
```
Comment (aturon) [inline _posts/2016-01-21-Rust-1.6.md]: A link to the docs page on this would be great.

**Comment by aturon** [comment]
Left a few notes, but otherwise LGTM!

File: _posts/2016-01-21-Rust-1.6.md (hunk 1 above)
Comment (steveklabnik) [inline _posts/2016-01-21-Rust-1.6.md]: I was also thinking about the float issue, but yeah, this is still better :+1:

**Comment by steveklabnik** [comment]
Updated!

**Pull request merged**